ts/src/pro/coinex.ts

```diff
@@ -711,23 +866,51 @@ export default class coinex extends coinexRest {
     }
 
     async watchOrders (symbol: Str = undefined, since: Int = undefined, limit: Int = undefined, params = {}): Promise<Order[]> {
+        /**
+         * @method
+         * @name coinex#watchOrders
+         * @description watches information on multiple orders made by the user
+         * @see https://docs.coinex.com/api/v2/spot/order/ws/user-order
+         * @see https://docs.coinex.com/api/v2/futures/order/ws/user-order
+         * @param {string} symbol unified market symbol of the market orders were made in
+         * @param {int} [since] the earliest time in ms to fetch orders for
+         * @param {int} [limit] the maximum number of order structures to retrieve
+         * @param {object} [params] extra parameters specific to the exchange API endpoint
+         * @param {bool} [params.trigger] if the orders to watch are trigger orders or not
+         * @returns {object[]} a list of [order structures]{@link https://docs.ccxt.com/#/?id=order-structure}
+         */
         await this.loadMarkets ();
+        const stop = this.safeBool2 (params, 'trigger', 'stop');
+        params = this.omit (params, [ 'trigger', 'stop' ]);
         await this.authenticate (params);
         let messageHash = 'orders';
         let market = undefined;
+        let marketList = undefined;
         const [ type, query ] = this.handleMarketTypeAndParams ('watchOrders', market, params);
-        const message: Dict = {
-            'method': 'order.subscribe',
-            'id': this.requestId (),
-        };
         if (symbol !== undefined) {
             market = this.market (symbol);
             symbol = market['symbol'];
-            message['params'] = [ market['id'] ];
+            marketList = [ market['id'] ];
             messageHash += ':' + symbol;
         } else {
-            message['params'] = [];
+            marketList = [];
+            if (type === 'spot') {
+                messageHash += ':spot';
```

Comment:
@Dan-krm you added `:spot` and `:swap` but you didn't update the messageHash inside `handleOrder` so right now this is not working

Reply:
it looks like you updated `handleOrders` and `handleMyTrades` for this so I'm going to mark these two reviews as resolved